### PR TITLE
DOCSUP-6212: Document the log_comment setting

### PR DESCRIPTION
**Pull request by gyuton**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #19583

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -769,6 +769,39 @@ Example:
...
+**Example**
+
+Query:
+
+``` sql
+SET log_comment = 'log_comment test', log_queries = 1;
+SELECT 1;
+SYSTEM FLUSH LOGS;
+SELECT type, query FROM system.query_log WHERE log_comment = 'log_comment test' AND event_date >= yesterday() ORDER BY event_time DESC LIMIT 2;
+WHERE event_date >= yesterday() ORDER BY event_time DESC LIMIT 2;
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:793]:
```suggestion
SELECT type, query FROM system.query_log WHERE log_comment = 'log_comment test' AND event_date >= yesterday() ORDER BY event_time DESC LIMIT 2;
```

Comment (olgarev) [inline docs/en/operations/settings/settings.md:793]:
```suggestion
SELECT type, query FROM system.query_log WHERE log_comment = 'log_comment test' AND event_date >= yesterday() ORDER BY event_time DESC LIMIT 2;
```

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -769,6 +769,39 @@ Example:
 log_query_threads=1
 ```
 
+## log_comment {#settings-log-comment}
+
+Logs the comment into [system.query_log](../system-tables/query_log.md) table and server log.
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:774]: Specifies the value for the `log_comment` field of the [system.query_log](../system-tables/query_log.md) table and comment text for the server log.

File: docs/ru/operations/settings/settings.md [hunk 3]
```diff
@@ -759,6 +759,38 @@ log_queries_min_type='EXCEPTION_WHILE_PROCESSING'
 log_query_threads=1
 ```
 
+## log_comment {#settings-log-comment}
+
+Указывает значение поля `log_comment` таблицы [system.query_log](../system-tables/query_log.md) и текст комментария в логе сервера.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:764]:
```suggestion
Задаёт значение поля `log_comment` таблицы [system.query_log](../system-tables/query_log.md) и текст комментария в логе сервера.
```

File: docs/ru/operations/settings/settings.md [hunk 4]
```diff
@@ -759,6 +759,38 @@ log_queries_min_type='EXCEPTION_WHILE_PROCESSING'
...
+
+Может быть использована для улучшения читабельности логов сервера. Кроме того, можно быстро выделить связанные с тестом запросы из `system.query_log` во время проведения [clickhouse-test](../../development/tests.md)
+
+Возможные значения:
+
+-   Любая строка не длиннее [max_query_size](#settings-max_query_size). При привышении длины, сервер сгенерирует исключение.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:770]:
```suggestion
-   Любая строка не длиннее [max_query_size](#settings-max_query_size). При превышении длины сервер сгенерирует исключение.
```

File: docs/ru/operations/settings/settings.md [hunk 5]
```diff
@@ -759,6 +759,38 @@ log_queries_min_type='EXCEPTION_WHILE_PROCESSING'
...
+
+Значениее по умолчанию: пустая строка.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:772]:
```suggestion
Значение по умолчанию: пустая строка.
```

**Comment by gyuton** [comment]
@CurtizJ Hi! Sorry about failed check, didn't see it. Now it's OK.

**Review by CurtizJ — APPROVED** [review]

**Pull request merged**